packages/api/src/routers/article_router.ts

```diff
@@ -93,16 +74,16 @@ export function articleRouter() {
   })
 
   router.get(
-    '/:id/:outputFormat/:priority/:voice?',
+    '/:id/:outputFormat/:priority?/:voice?/:secondaryVoice?',
```

Comment:
I wonder if we should make some of the optional stuff query params, it might be more clear to have `/:id/:outputFormat?voice=Jenny...&secondaryVoice=Guy..&priority=high`

Reply:
Yes, it would be def better

Reply:
@jacksonh Updated